### PR TITLE
release v1.0.0

### PR DESCRIPTION
**Pull request by weekface**
### Check List 

Tests 

 - No code

Code changes

 - Has documents change

Side effects

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:
 
 ```release-note
NONE
 ```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,180 @@
...
+  ...
+```
+
+`monitor` section in new `values.yaml`:
+
+```
+monitor:
+  create: true
+   initializer:
+     image: pingcap/tidb-monitor-initializer:v3.0.0
```
Comment (tennix) [inline CHANGELOG.md]:
```suggestion
     image: pingcap/tidb-monitor-initializer:v3.0.1
```

File: README.md [hunk 2]
```diff
@@ -44,23 +44,13 @@ Read the [Roadmap](./ROADMAP.md).
 
 ## Quick start
 
-Choose one of the following tutorials:
+Read the [Quick Start Guide](https://pingcap.com/docs/v3.0/tidb-in-kubernetes/tidb-operator-overview/), which includes all the guides for managing TiDB clusters in Kubernetes.
```
Comment (DanielZhangQD) [inline README.md]: Suggest changing to "all the quick start guides"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,180 @@
...
+### Stability tests cases added
+
+- stop all etcds and kubelets
+
+### A lot of improvements
+
+- Simplify GKE SSD setup
+- Modularization for AWS terraform scripts
+- turn on automatic failover feature by default
+- Configmap rollout is now enabled by default
```
[the remark this answers is not in the document]
Comment (weekface) [reply CHANGELOG.md]: this is `v1.0.0-beta.3` feature

File: README.md (hunk 2 above)
Comment (weekface) [reply README.md]: copy from tidb: https://github.com/pingcap/tidb#quick-start

File: CHANGELOG.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tennix) [reply CHANGELOG.md]: This is enabled by default in since last release.

[the remark this answers is not in the document]
Comment (weekface) [reply CHANGELOG.md]: Done.

**Pull request merged**